python/tools/whisper/transcribe.py

```
@@ -0,0 +1,167 @@
+
+def model_cache_present(model: str) -> bool:
+    """Check whether the given model is already downloaded in the cache volume.
+
+    Args:
+        model: faster-whisper model name (e.g. ``large-v3``).
+
+    Returns:
+        True if the HuggingFace cache directory for the model exists in the volume.
+    """
+    cache_directory = f"hub/models--Systran--faster-whisper-{model}"
+    completed = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)